src/pages/auth/UserRegister.tsx

```
@@ -48,7 +48,7 @@ export default function UserRegister() {
     gender: "",
   });
 
-  const validateField = (field: any, value: any) => {
+  const validateField = (field: any, value: any) => { //NOSONAR
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)